slides/03-what-makes-a-model.qmd

```diff
@@ -401,19 +414,6 @@ workflow() %>%
 
 ## A model workflow `r hexes("parsnip", "workflows")`
 
-```{r}
-tree_spec <-
-  decision_tree() %>% 
-  set_mode("regression")
-
-workflow() %>%
-  add_variables(outcomes = latency, predictors = everything()) %>%
-  add_model(tree_spec) %>%
-  fit(data = frog_train) 
-```
-
-## A model workflow `r hexes("parsnip", "workflows")`
-
```

Comment:
So we removed this one but not the `add_formula(latency ~ .) %>%` one from above?

Reply:
I believe Max said to keep this so we teach folks both `workflow(preprocessor, spec)` and `workflow() %>% add_formula(form) %>% add_model(spec)`.